Modules/Devices/src/Devices.Application/Identities/Commands/CancelDeletionProcess/CancelDeletionProcessResponse.cs

```diff
@@ -0,0 +1,19 @@
+﻿using Backbone.DevelopmentKit.Identity.ValueObjects;
+using Backbone.Modules.Devices.Domain.Entities.Identities;
+
+namespace Backbone.Modules.Devices.Application.Identities.Commands.CancelDeletionProcess;
+public class CancelDeletionProcessResponse
+{
+    public CancelDeletionProcessResponse(IdentityDeletionProcess deletionProcess)
+    {
+        Id = deletionProcess.Id;
+        Status = deletionProcess.Status;
+        CancelledAt = deletionProcess.CancelledAt ?? throw new Exception($"The '{nameof(IdentityDeletionProcess.CancelledAt)}' property of the given deletion process must not be null.");
```

Comment:
It's debatable. The `CancelledAt` property theoretically is nullable. But since the CancelDeletionProcessResponseCommand always sets it, it's never null when creating the response. 
I think I would go with a null suppression in this case.

Reply:
What should we do with `ApproveDeletionProcessResponse` then, it's the same there:
```csharp
        ApprovedAt = deletionProcess.ApprovedAt ?? throw new Exception($"The '{nameof(IdentityDeletionProcess.ApprovedAt)}' property of the given deletion process must not be null.");
        ApprovedByDevice = deletionProcess.ApprovedByDevice ?? throw new Exception($"The '{nameof(IdentityDeletionProcess.ApprovedByDevice)}' property of the given deletion process must not be null.");
```

I'm fine with either way, I just think we should be uniform about it.

Reply:
Also in `StartDeletionProcessAsOwnerResponse`
```csharp
   ApprovedAt = deletionProcess.ApprovedAt ?? throw new Exception($"The '{nameof(IdentityDeletionProcess.ApprovedAt)}' property of the given deletion process must not be null.");
   ApprovedByDevice = deletionProcess.ApprovedByDevice ?? throw new Exception($"The '{nameof(IdentityDeletionProcess.ApprovedByDevice)}' property of the given deletion process must not be null.");
```

Reply:
Hmm. Okaaay, then let's keep throwing the exception. 😄